projects/styles/lf-laserfiche-lite/lf-laserfiche-lite.scss

```diff
@@ -7,3 +7,4 @@
 @use "sass/rwc-tree-lf";
 @use "sass/rwc-checklist-lf";
 @use "sass/material";
+@use "../../../node_modules/@angular/material/legacy-prebuilt-themes/legacy-indigo-pink.css";
```

Comment:
add this also in the lf-ms-office-lite.scss

change the documentation where we ask the user to import indigo-pink in their index.html (for example:   https://github.com/Laserfiche/lf-ui-components/blob/15aa10b18a3fbb2f7977e2bfce844f91a695ce77/projects/lf-documentation/src/app/getting-started/getting-started.component.html#L78C1-L78C1)